src/helpers/get-static-urls-by-environment.js

```diff
@@ -2,16 +2,16 @@ import { getEnvironment } from './get-environment'
 
 const urls = {
   manager: {
-    stage: 'https://stage-manager.azion.com',
-    production: 'https://manager.azion.com'
+    stage: 'https://stage-console.azion.com',
+    production: 'https://console.azion.com'
   },
   managerEOL: {
```

Comment:
We should probably remove these, as well as the code referencing `manager` and `managerEOL`.

Reply:
Yes, I will remove it.